mycroft/client/enclosure/__init__.py

```diff
@@ -77,7 +77,8 @@ def read(self):
                 LOG.error("Reading error: {0}".format(e))
 
     def process(self, data):
-        self.ws.emit(Message(data))
+        if not "mycroft.stop" in data:
```

Comment:
Let's add a comment here, like:
# TODO: Look at removing this altogether.  Review if any Arduino serial messages are handled by other messagebus subscribers